pom.xml

```diff
@@ -240,7 +240,7 @@
         <version.io.opentracing.jaxrs2>0.1.7</version.io.opentracing.jaxrs2>
         <version.io.opentracing.tracerresolver>0.1.5</version.io.opentracing.tracerresolver>
         <version.io.opentracing.servlet>0.1.0</version.io.opentracing.servlet>
-        <version.io.reactivex.rxjava>2.1.16</version.io.reactivex.rxjava>
+        <version.io.reactivex.rxjava>2.2.2</version.io.reactivex.rxjava>
```

Comment:
@asoldano Please approve, since it looks like Resteasy uses this.

Reply:
@bstansberry approved, we're currently testing with version 2.2.2 on RESTEasy master